gulp/tasks/styles.js

```diff
@@ -14,7 +13,7 @@ module.exports = function (gulp, $, config) {
         extensions: ['.scss']
       }))
       .pipe($.sourcemaps.init())
-      .pipe($.sass({includePaths: [bowerFiles]}))
+      .pipe($.sass({includePaths: []}))
```

Comment:
that's the idea, yes. You explicitly require the styles you want however, in your index style file, as we do now.